docs/recipes/template.rst

```diff
@@ -0,0 +1,36 @@
+Recipe Title
+============
+
+.. use = for sections, ~ for subsections and - for subsubsections
+
+**Feature available since version:** <version_num>
+
+General description here.
+
+1. Step 1
+   make sure you indent correctly inside steps
```

Comment:
Sometimes, I hate RST... I have so often had to recompile docs because I am missing on single space. :-(